dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/ProjectService.java

```diff
@@ -204,4 +205,27 @@ Result queryProjectWithAuthorizedLevelListPaging(Integer userId, User loginUser,
      */
     Result queryAllProjectListForDependent();
 
+    /**
+     * query all workflow count
+     * @param loginUser login user
+     * @return workflow count
+     */
+    Result queryAllWorkflowCounts(User loginUser);
```

Comment:
A question here, should we make three new interface return the same type? like a more general type about statistics data type for `List[Map<String, Long>]`?

Reply:
done.